ext/gd/php_gd.h

```diff
@@ -55,6 +55,12 @@
 #define PHP_IMG_TGA  128
 #define PHP_IMG_AVIF 256
 
+#ifdef HAVE_GD_BUNDLED
+GD_BUNDLED 1
+#else
+GD_BUNDLED 0
+#endif
```

Comment:
```suggestion
#ifdef HAVE_GD_BUNDLED
# define GD_BUNDLED 1
#else
# define GD_BUNDLED 0
#endif
```

Reply:
Yeah, I've just noticed it as well :) Unfortunately, I forced pushed it (I wouldn't think about such a quick review)